orbit/pkg/packaging/wix_template.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Product
+    Id="*"
+    Name="Fleet osquery"
+    Language="1033"
+    Version="{{.Version}}"
+    Manufacturer="Fleet Device Management (fleetdm.com)"
+    UpgradeCode="B681CB20-107E-428A-9B14-2D3C1AFED244" >
+
+    <Package
+      Keywords='Fleet osquery'
+      Description="Fleet osquery"
+      InstallerVersion="500"
+      Compressed="yes"
+      InstallScope="perMachine"
+      InstallPrivileges="elevated"
+      Languages="1033" />
+
+    <Property Id="REINSTALLMODE" Value="amus" />
+
+    <!-- 
+      Cleanup any existing local database (to support downgrades and also to start fresh). 
+      See https://www.hass.de/content/wix-how-use-removefolderex-your-xml-scripts.
+    -->
+    <!--
+      RemoveFolderEx requires that we "remember" the path for uninstall.
+      Read the path value and set the OSQUERYDBFOLDER property with the value.
+    -->
+    <Property Id="OSQUERYDBFOLDER">
```

Comment:
any thoughts on removing the whole folder, which also fixes https://github.com/fleetdm/fleet/issues/3563?